website/frontend/src/common/components/Select/Select.module.scss

```diff
@@ -0,0 +1,26 @@
+@use 'assets/scss/variables' as *;
+
+.label {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  font-size: 16px;
+  font-weight: 300;
+  color: $textColorSecondary;
+}
+
+.text {
+  margin-right: 16px;
+}
+
+.select {
+  flex: 1;
+  padding: 10px 45px 10px 20px;
+  color: $textColor;
+  background: #282828 url('./images/arrow.svg') center right 16px no-repeat;
```

Comment:
you could inline this image by using [this tool](https://yoksel.github.io/url-encoder/), and reduce extra symbols from svg by using [this tool](https://jakearchibald.github.io/svgomg/)